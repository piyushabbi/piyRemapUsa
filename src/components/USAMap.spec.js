import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import USAMap from './USAMap';

describe('USAMap Component', () => {
	let wrapper;

	beforeEach(() => {
		const titleHandler = jest.fn();
		wrapper = shallow(<USAMap />);
	});

	it('renders snapshot correctly', () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});

	it('executes mouse event', () => {
		// console.log(wrapper.instance())
	});
});
