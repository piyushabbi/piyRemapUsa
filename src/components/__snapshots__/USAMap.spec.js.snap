// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`USAMap Component renders snapshot correctly 1`] = `
<Fragment>
  <svg
    className="us-state-map"
    height={593}
    viewBox="0 0 959 593"
    width={959}
    xmlns="http://www.w3.org/2000/svg"
  >
    <title>
      Yo Man! Wadd up......
    </title>
    <g
      className="outlines"
    />
  </svg>
</Fragment>
`;
